test/e2e/__init__.py

```diff
@@ -372,3 +374,15 @@ def get_sagemaker_pipeline(pipeline_name: str):
             f"SageMaker could not find a pipeline with the name {pipeline_name}. Error {error}"
         )
         return None
+
+
+def delete_custom_resource(
+    reference,
+    wait_period=cfg.JOB_DELETE_WAIT_PERIODS,
+    wait_length=cfg.JOB_DELETE_WAIT_LENGTH,
+):
+    deleted = True
+
+    if k8s.get_resource_exists(reference): 
+        _, deleted = k8s.delete_custom_resource(reference, wait_period*2, wait_length)
```

Comment:
nit: lets double the value of the constant or at the passing method to keep the method generic 

lets use `cfg.DELETE_WAIT_PERIOD, cfg.DELETE_WAIT_LENGTH` for the ones with 3, 10? similar to job